### ISSUE TITLE
Connection Problems after a time

### ISSUE THREAD

**Issue by Tockra** [jabbink/PokemonGoBot#1058]
**Description:**
After a time running the bot (1-3 hours) the bot doesn't use pokestops and find pokemon. I think it hasn't a connection anymore, because the console throws a exception (it doesn't appear in the log).

After a restart everything works well !

**Steps to reproduce:**

1. Start Bot with TC
2. Run a time

**Expected behavior:**
You know ;)

**Actual behavior:**
Bot doesn't run after a while

**Stacktrace (If it's a crash):**
http://pastebin.com/nFZKz9s0

I dont have the whole stacktrace because I cant scroll in my console :( : 
http://pastebin.com/k9xbYk8t

**Version:**
db5d8e20ab422c8cc0dca2ded967691551b22a6c

**Operating System:**
Latest Debian

**Java version:**
java version "1.8.0_101"
Java(TM) SE Runtime Environment (build 1.8.0_101-b13)
Java HotSpot(TM) 64-Bit Server VM (build 25.101-b13, mixed mode)

**Comment by Sadfaffel** [comment]
Yeah, i have same thing too but i use windows and i made .bat file for restarting it every 90mins.
Dont know anything about Debian tho.

**Comment by HoneyWheat** [comment]
see #680

**Comment by Tockra** [comment]
I dont know if its the same. The Stacktraces are different. And the error appears since last night...

**Comment by patzor** [comment]
have u try running ./gradlew clean build

and try again if it does the same error? I got that same error i just did that..
